ptx/S_12_Div_Thm.ptx

```diff
@@ -16,91 +16,323 @@
         </p>
       </statement>
     </task>
     <task xml:id="DVT_verify_task5">
       <statement>
-        <p> Give inequalities to describe the interior of the closed surface S_1. </p>
+        <p> Set up and compute the triple integral for <m>\iiint_Q div(\vF) dV</m>. Hint: use cylindrical coordinates.</p>
       </statement>
     </task>
     <task xml:id="DVT_verify_task6">
       <statement>
-        <p> Integrate the divergence over the interior of S_1. </p>
+        <p>Verify that your answers for <xref ref="DVT_verify_task3">part</xref> and <xref ref="DVT_verify_task6">part</xref> are the same and thus that the Divergence Theorem holds for this example.</p>
       </statement>
     </task>
     <task xml:id="DVT_verify_task7">
       <statement>
-        <p> Show that the net flux through S_1 is exactly the same as the integral of the divergence on the interior of S_1. </p>
+        <p>Was the left hand side or right hand side of the Divergence Theorem more tedious to calculuate in this example? Do you think this will be true for most other cases where the divergence theorem applies?</p>
       </statement>
     </task>
   </activity>
 </subsection>
+<subsection>
+  <title>Summary</title>
   <p>
-    Exercises will include several purely calculation based questions
-  </p>
-
-  <p>
-    Should we have a subsection at the end of each section that shows how to do the relevent graphs and computations in Sage or Calcplot3d?
+  <ul>
+    <li>
+      <p>
+        The flux of a continous vector field through a closed surface can be computed by integrating the divergence of the vector field over the interior of the closed surface.
+      </p>
+    </li>
+  </ul>
   </p>
+</subsection>
+<exercises>
+  <exercise>
+    <statement>
+        <p>
+          Let <m>Q</m> be the volume enclosed by <m>x=0</m>, <m>x=1</m>, <m>y=0</m>, <m>y=1</m>, <m>z=0</m>, and <m>z=1</m>. Compute the flux of <m>\vF = \langle z^2-xy,4yz+cos(x/\pi),e^{xyz} \rangle</m> through each of the six cube faces.
+        </p>
+    </statement>
+    <hint><p>A hint! Don't feel obligated to use hints, but they are an option.</p></hint>
+    <answer>
+        <p>
+            
+        </p>
+    </answer>
+    <solution>
+        <p>
+            The full solution. Full details go here.
+        </p>
+    </solution>
+  </exercise>
+  <exercise>
+    <statement>
+      <p>
+        Explain why the flux of <m>\vF=\langle y,-z,x\rangle</m> through the unit sphere is zero. You should not compute any integrals as part of your explanation.
+      </p>
+    </statement>
+  </exercise>
+  <exercise>
+    <statement>
+      <p>Let <m>Q</m> be the volume given by <m> 0\geq z 3</m>, <m>1\geq r \geq 2</m>, and <m>0\geq \theta < 2 \pi</m>. Give an example of a vector field with component functions that are linear in <m>x</m>, <m>y</m>, and <m>z</m> such that the flux of the vector field through the boundary of <m>Q</m> is 17.</p>
```

Comment:
@nelong There's a bit of a mess here in the definition of Q. I'm seeing some \geq that probably are supposed to be \leq? And then you can't use a naked `<` in XML, so if you need one, PTX supplies a \lt macro by default.

Reply:
I just missed this stuff. I think everything is fixed in these issues.